backend/app/utils/background_tasks.py

```
@@ -499,9 +535,79 @@ async def _process_content_async(self, content_id: str, user_id: str):
+                                    session.add(content_item)
+
+                                    # 通知前端AI处理失败
+                                    try:
+                                        await content_event_manager.notify_content_status(
+                                            user_id=user_id,
+                                            content_id=content_id,
+                                            status="processing",
+                                            progress=75,
+                                            title=content_item.title or "新内容",
+                                            error=f"AI分析失败: {str(preprocessing_err)}",
+                                        )
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'error' in method call

(E1123)